packages/shared/src/hooks/useSquadChecklist.tsx

```diff
@@ -151,60 +152,58 @@ const useSquadChecklist = ({
           component: (props) => <GoPublicStep {...props} squad={squad} />,
         },
         actions: getPublicActions(squad),
+        condition: () => {
+          const isGoPublicDismissed =
+            isChecklistReady &&
+            checkHasCompleted(ActionType.HidePublicSquadStep);
+          return !squad.public && !isGoPublicDismissed;
+        },
       }),
-    };
-
-    if (isPushSupported) {
-      step[ActionType.EnableNotification] = createChecklistStep({
+      [ActionType.HidePublicSquadAbout]: createChecklistStep({
+        type: ActionType.HidePublicSquadAbout,
+        step: {
+          title: 'Learn about Squads',
+          description: `Learn the benefits of a public squad and how to expand your community. Good luck!`,
+          component: (props) => <PublicAboutStep {...props} />,
+        },
+        actions,
+        condition: () => {
+          return !!squad.public;
```

Comment:
Should also only be for admins? (not sure if this whole section maybe is already)

Reply:
I put this step only in the list for admins ([you can check here](https://github.com/dailydotdev/apps/pull/3489/files#diff-c6a0631a6e91d8c7c9e15687f2859e1614603ee0c6a48a00fd38de37166e0eb3R51)), so shouldn't be visible for users ✅

Reply:
Ah perfect then :D